resources/base/deployment.yaml

```diff
@@ -40,7 +40,7 @@ spec:
             - name: MINIO_CONSOLE_TLS_ENABLE
               value: "off"
             - name: OPERATOR_STS_ENABLED
-              value: "off"
+              value: "on"
```

Comment:
what is the impact?

Reply:
STS webserver will start when operator containers start

Reply:
Okay so by default this service will be running, and it's public facing ?

Reply:
Is kubernetes cluster internal, service type is ClusterIP, will not be exposed oustide the cluster unless explictly modified the service.

https://github.com/minio/operator/blob/6c3d60899715c7c19d15773e8310d97e17508e11/resources/base/service.yaml#L19-L33

The webserver is the STS endpoint where k8s apps (pods) will ask for IAM credenials.
https://github.com/minio/operator/blob/master/docs/STS.md

Policy-serviceaccount relation to provide sts credentials is managed by the [PolicyBinding](https://github.com/minio/operator/blob/master/docs/policybinding_crd.adoc) resource.

All that the `OPERATOR_STS_ENABLED =on` flag does is enable the webserver that handle the sts calls